hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/txns/crypto/ApproveAllowanceLogic.java

```diff
@@ -101,14 +101,16 @@ private void applyCryptoAllowances(final List<CryptoAllowance> cryptoAllowances,
             final var cryptoMap = accountToApprove.getMutableCryptoAllowances();
 
             final var spender = Id.fromGrpcAccount(allowance.getSpender());
-            accountStore.loadAccountOrFailWith(spender, INVALID_ALLOWANCE_SPENDER_ID);
-
             final var amount = allowance.getAmount();
 
             if (cryptoMap.containsKey(spender.asEntityNum()) && amount == 0) {
+                // spender need to be validated as being a valid account when removing allowances,
```

Comment:
`// spender need NOT be validated...` here?

Reply:
ah yes 😞